drivers/docker/config_test.go

```diff
@@ -41,6 +42,32 @@ func TestConfig_ParseHCL(t *testing.T) {
 	}
 }
 
+func TestConfig_ParseJSON_Regression(t *testing.T) {
+	cases := []struct {
+		name  string
+		input map[string]interface{}
```

Comment:
I think the test would be much better if we specify what the expected resulting values are, not just that it parses without errors.